Gemfile.lock

```diff
@@ -137,7 +137,7 @@ GEM
     ast (2.0.0)
     astrolabe (1.3.0)
       parser (>= 2.2.0.pre.3, < 3.0)
-    autoprefixer-rails (5.2.0)
+    autoprefixer-rails (6.0.3)
```

Comment:
autoprefixer isn't one of our packages and it's a full version upgrade, is this needed by one of the other tools?